apps/web/src/server/applications/case/representations/representation-details/representation-status/representation-status.view-model.js

```diff
@@ -0,0 +1,124 @@
+import { url } from '../../../../../lib/nunjucks-filters/url.js';
+
+/**
+ * @typedef {object} Representation
+ * @property {number} id
+ * @property {string} reference
+ * @property {string} status
+ * @property {Contact[]} contacts
+ * @property {boolean} redacted
+ * @property {string} received
+ * @property {string} originalRepresentation
+ * @property {string} representationExcerpt
+ * @property {string} redactedRepresentation
+ * @property {string} redactedRepresentationExcerpt
+ * @property {string} redactedNotes
+ * @property {string} redactedNotesExcerpt
+ * @property {string} redactedBy
+ */
+
+/**
+ * @typedef {object} Contact
+ * @property {string|null} firstName
+ * @property {string|null} lastName
+ * @property {string} organisationName
+ * @property {string|null} contactMethod
+ * @property {string} type
+ * @property {string|null} jobTitle
+ * @property {boolean} under18
+ * @property {string|null} email
+ * @property {string|null} phoneNumber
+ * @property {object} address
+ */
+
+/**
+ * @param {string} caseId
+ * @param {string} representationId
+ * @returns { string }
+ */
+export const getPreviousPageUrl = (caseId, representationId) =>
+	url('representation-details', {
```

Comment:
not sure why you would use the nunjucks filters here ? 

That is filters for the render.

Reply:
this is inconsistent with how the other back links are managed - see redact rep and change.

Reply:
I've used it to _be_ consistent with the redact page, something to discuss then

Reply:
i don't think redact should be doing that either. Must have missed on it's PR.